admin/jobs/river/river.go

```diff
@@ -339,9 +339,10 @@ func (c *Client) InitOrgBilling(ctx context.Context, orgID string) (*jobs.Insert
 	}, nil
 }
 
-func (c *Client) RepairOrgBilling(ctx context.Context, orgID string) (*jobs.InsertResult, error) {
+func (c *Client) RepairOrgBilling(ctx context.Context, orgID, biller string) (*jobs.InsertResult, error) {
 	res, err := c.riverClient.Insert(ctx, RepairOrgBillingArgs{
-		OrgID: orgID,
+		OrgID:  orgID,
+		Biller: biller,
```

Comment:
This is passed only for deduplication, lets say if by mistake we triggered repair job with noop biller then triggering again after changing config to Orb biller would reject the job as duplicate. Thats why added this, its not used for anything else.

Reply:
Although repair job is idempotent as of now and we can just remove duplication check in job submission logic but this seems more efficient to do.

Reply:
It can happen when the code goes live and for whatever reason the orb keys were missed or wrong config name used so noop biller will be configured and someone triggered repair job.

Reply:
Just worried about the above case - so either we do this or we don't deduplicate jobs or if it happen then we will have to manually delete jobs from river UI and trigger repair job again.

Reply:
For the noop biller, wont the job complete and be removed immediately? I guess I don't understand how it would end up running for a long time.

The case of the biller being removed in production would also be a catastrophic error I think with much worse implications than this? (As an aside, maybe we should crash loop if the biller is missing in a deployment where there previously was a biller?)

Reply:
River deduplicates with completed job as well, if it ran in past but has same job Args then it is treated as duplicated unless the job state is cancelled or errored.

I am worried about the first run only where we will trigger this job not subsequent run where it won't be needed at all actually.

Reply:
May be just allowing duplicate jobs to run will remove the confusion and no need to biller arg. Anyways, job runs repair job only for orgs which has empty billing customer id so that will be fine.